spyder/plugins/tours/container.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+# -----------------------------------------------------------------------------
+# Copyright (c) 2009- Spyder Project Contributors
+#
+# Distributed under the terms of the MIT License
+# (see spyder/__init__.py for details)
+# -----------------------------------------------------------------------------
+
+"""
+Tours Container.
+"""
+
+from collections import OrderedDict
+
+# Local imports
+from spyder.api.exceptions import SpyderAPIError
+from spyder.api.translations import get_translation
+from spyder.api.widgets.main_container import PluginMainContainer
+from spyder.plugins.tours.tours import TourIdentifiers
+from spyder.plugins.tours.widgets import AnimatedTour, OpenTourDialog
+
+# Localization
+_ = get_translation('spyder')
+
+# Set the index for the default tour
+DEFAULT_TOUR = TourIdentifiers.IntroductionTour
+
+
+class TourActions:
+    """
+    Tours actions.
+    """
+
+    ShowTour = "show tour"
+
+# --- Plugin
+# ----------------------------------------------------------------------------
+
+
+class ToursContainer(PluginMainContainer):
+    """
+    Tours container.
+    """
+
+    def __init__(self, name, plugin, parent=None):
+        super().__init__(name, plugin, parent=parent)
+
+        self._main = plugin.main
+        self._tours = OrderedDict()
+        self._tour_titles = OrderedDict()
+        self._tour_widget = AnimatedTour(self._main)
+        self._tour_dialog = OpenTourDialog(
+            self, lambda: self.show_tour(DEFAULT_TOUR))
+        self.tour_action = self.create_action(
+            TourActions.ShowTour,
+            text=_("Show tour"),

```

Comment:
```suggestion
            text=_("Show tour"),
            icon=self.create_action('tour'),
```

For this suggestion to work, you need to first add the `tour ` key to the icon_manager dict.